index.html

```diff
@@ -94,19 +104,25 @@ <h1 class="tit-projects">Projects</h1>
          </div>            
       </article>
    </section>
+
+   <section class="sec-works-pc">
+      <img class="img-works-pc" src="resources/Group-117.png" alt="works-projects">
+   </section>
```

Comment:
- [ ] Please note that you are to build this section with HTML and CSS. I noticed you placed an image in your desktop version instead of building it. Kindly correct this, please.🙃